routes/api/users.js

```diff
@@ -61,18 +61,20 @@ router.put("/:username",(request,response) =>{
      response.send({msg:"User deleted",data:newdata});
 })
 
-router.put("/:username/grades",(request,response) =>{
-    const x=request.body;
+router.put("/grades/:username",(request,response) =>{
+    const coursename=request.body.courseName;
+    const thegrade=request.body.grade;
```

Comment:
the variable name must be the same as the one in the body to be understandable throw the code.
Change `thegrade` to be `grade`